driver-async/src/main/com/mongodb/async/client/FindIterableImpl.java

```diff
@@ -99,6 +99,11 @@
         return this;
     }
 
+    @Override
+    public Integer getBatchSize() {
```

Comment:
Given the danger, please add a test for getBatchSize for FindIterable and at least one other MongoIterable subclass.

Reply:
Done